### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by W4ngatang**
Fixes some bugs:

- In some task, accidentally wrote labels instead of predictions
- fix bug in write_preds test where we didn't have predictions...
- fix preprocessing bug in broad coverage diagnostics
- use `rte-superglue` classifier for Winogender
- fix bug in Winogender metrics that used logits as predictions
- fix bug in Winogender preprocessing where there were no labels when using BERT
- fix bug in download script where we weren't unzipping downloads
- add v2.0 to all download URLs and update broad coverage diagnostic

Some cleanup:

- remove mention of wmt in evaluation
- update baseline script comment

### CODE REVIEW

**Comment by W4ngatang** [comment]
Failing tests until Amanpreet moves the data to the v2.0 URLs.

**Comment by W4ngatang** [comment]
It'd be really nice is someone could sanity check this and run the diagnostics from this branch.

File: scripts/download_superglue_data.py [hunk 1]
```diff
@@ -51,12 +51,17 @@ def download_diagnostic(data_dir):
     diagnostic_dir = os.path.join(data_dir, "RTE", "diagnostics")
```
Comment (sleepinyourhat) [inline scripts/download_superglue_data.py]: Seems cleaner to make these two different tasks/methods, but not a big deal.

**Comment by pruksmhc** [comment]
There's some path-related issues with the current download_scripts

**Comment by pruksmhc** [comment]
Basically, the output of python download_superglue_scripts.py for RTE and diagnostics is AX-B and AX-g zip files in the same level as RTE.
